src/api-engine/api/lib/peer/chaincode.py

```diff
@@ -315,28 +319,26 @@ def invoke(self, orderer_url, orderer_tls_rootcert, channel_name, cc_name, args,
                 invoke_command_tls = "{} chaincode invoke -o {} --tls --cafile {} --channelID {} --name {} -c '{}'"
 
             if os.getenv("CORE_PEER_TLS_ENABLED") == "false" or os.getenv("CORE_PEER_TLS_ENABLED") is None:
-                if self.version in BasicEnv.binary_versions_v2:
```

Comment:
Remove `BasicEnv.binary_versions_v2` conditional check because it isn't initialized before